.vscode/settings.json

```diff
@@ -32,5 +32,17 @@
   },
   "go.useLanguageServer": true,
   "go.lintOnSave": "off",
-  "go.vetOnSave": "off"
+  "go.vetOnSave": "off",
+  "sqltools.connections": [
```

Comment:
i dont think these were meant to get in this PR? seems like its not a common connection for all the people in the org